.gitignore

```diff
@@ -6,4 +6,4 @@
 .settings/
 target/
 README.md.bak
-
+*.versionsBackup
```

Comment:
What is this for?

Reply:
The maven plugin that updates release versions creates backups of POM files with this extension.  Just avoiding having those checked in by accident.